### ISSUE TITLE
Core bumped (Unable to initialize backend 'tpu_driver, CUDA_ERROR_OUT_OF_MEMORY,device CUDA:0 not supported by XLA service, etc..)

### ISSUE THREAD

**Issue by chersiakingdom** [deepmind/alphafold#346]
Hi. First of all, thank you for your help.
No matter how hard I searched, I couldn't find any information to fix this error.
(Even though it is a problem that many people are struggling with, there is no solution.)

Below is the error message.

```
Unable to initialize backend 'tpu_driver' :NOT_FOUND: 
Unable to find driver in registry given worker:
~~
unable to create StreamExecutor for CUDA:0: failed initializing StreamExecutor for CUDA device ordinal 0: INTERNAL: faild call to ceDevicePrimaryCtxRetainL CUDA_ERROR_OUT_OF_MEMORY: out of memory; total memory reported: 2544170048
~~
Attemping to fetch value instead of handling error INVALID_ARGUMENT: device CUDA:0 not supported by XLA service
Fatal Python error: Aborted
~~~
9 Aborted (core bumped)
```

It works fine until Docker image build, but when I finally run the file, above error occurs.

Following is the information of my server.
1. Ubuntu 18.04.01
2. Conda 3.9.2
3. Nvidia-driver-465
4. CUDA 11.3
5. cuDNN 8.2.1
6. Pytorch 1.9.0

Additionally, alphafold data is stored on the HDD, 
and when installing, there is a problem with the jaxlib library version, so I modified line 68 of the dockerfile as follows. (not sure about line number)
 pip3 install ---upgrade jax jaxlib== 0.1.69 
to
pip3 install ---upgrade jax jaxlib=>=0.1.69
(== to >=)

Any help would be greatly appreciated.

**Comment by Augustin-Zidek** [comment]
This might be due to the same reasons as the problems you are seeing in https://github.com/deepmind/alphafold/issues/351 (i.e. might be fixed by the CUDA 11 -> 11.1 bump). Could you try running with v2.1.2?

**Comment by chersiakingdom** [comment]
All errors have been fixed with the new Alphafold v2.1.2, and it works fine on my computer. Thanks so much for the advice.
